CONTRIBUTING.md

```diff
@@ -55,15 +55,178 @@ Clone your repo fork to a local machine.
 git clone https://github.com/<UserName>/machine-learning.git
 ```
 
-#### Install Python & Conda
+ **There will be slight differences on different systems**
+
+## For Windows
```

Comment:
This should be in header3 `### For Windows`. And all its subsection should be in `####`